spec/index.bs

```diff
@@ -7960,6 +7960,25 @@ dictionary GPURenderPassColorAttachment {
         render pass. If not [=map/exist|provided=] defaults to `{r: 0, g: 0, b: 0, a: 0}`. Ignored
         if {{GPURenderPassColorAttachment/loadOp}} is not {{GPULoadOp/"clear"}}.
 
+        The members of {{GPURenderPassColorAttachment/clearValue}} are all double values, so
+        they will first be converted to the fully qualified format type of
+        {{GPURenderPassColorAttachment/view}} before being set as the clear value of
+        {{GPURenderPassColorAttachment/view}}.
+
+        Let |colorAttachmentFormat| be
+        {{GPURenderPassColorAttachment/view}}.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/format}}.
+        - If the components of |colorAttachmentFormat| are floating point values, the members of
+            {{GPURenderPassColorAttachment/clearValue}} will be converted into floating point
+            values.
+        - If the components of |colorAttachmentFormat| are signed or unsigned normalized values,
+            the members of {{GPURenderPassColorAttachment/clearValue}} will first be clamped
+            into the range [0, 1], then converted into floating point values, and finally
+            converted into the signed or unsigned normalized values.
+        - If the components of |colorAttachmentFormat| are signed or unsigned integer values,
+            the members of {{GPURenderPassColorAttachment/clearValue}} will first be clamped
+            into the range of signed or unsigned integer values with same bit length, and then
+            rounded into the nearest signed or unsigned integer values.
```

Comment:
needs to define rounding of "halfway" values - see WebIDL rules here:
https://webidl.spec.whatwg.org/#abstract-opdef-converttoint

Also need to decide whether we clamp values that are out of range (probably?) or do something else with them